lib/entities/space.ts

```diff
@@ -8,6 +8,7 @@ import enhanceWithMethods from '../enhance-with-methods'
 export type SpaceProps = {
   sys: BasicMetaSysProps & { organization: { sys: { id: string } } }
   name: string
+  defaultLocale?: string
```

Comment:
Should we have an extended type for `CreateSpaceProps` or are we sure wherever SpaceProps is used, we can set a `defaultLocale` param?

Reply:
Extension is correct. It's also used in `getSpace` and others that doesn't return `defaultLocale`